src/net/sourceforge/kolmafia/persistence/ItemDatabase.java

```diff
@@ -2208,6 +2208,36 @@ public static void parseFireExtinguisher(final String desc) {
     }
   }
 
+  public static void parseVampireVintnerWine(final String desc) {
+    String mod = DebugDatabase.parseItemEnchantments(desc, KoLConstants.CONSUME_DRINK);
+    Modifiers.overrideModifier("Item:[" + ItemPool.VAMPIRE_VINTNER_WINE + "]", mod);
+    String type = "";
+    switch (Modifiers.getStringModifier("Item", ItemPool.VAMPIRE_VINTNER_WINE, "Effect")) {
```

Comment:
Is this switch statement something we could configure in consequences.txt using DESC_ITEM entries?